flink/v1.14/flink-runtime/NOTICE

```diff
@@ -89,3 +89,14 @@ file:
 |
 | This product includes software developed at
 | The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+This binary artifact includes Apache HTTPComponents Client with the following
+in its NOTICE file:
```

Comment:
Of the dependency is another ASF project and has a boilerplate NOTICE file (like this one) we don't need to include it in the NOTICE file. We do need to make sure that we note it in the LICENSE file, though. Could you update this to add the standard LICENSE file content, @kbendick?

Reply:
Ok. Updated to make the LICENSE content mirror Apache Arrow.